geodepy/tests/resources/Test_Conversion_ISG_Grid.csv

```diff
@@ -0,0 +1,212 @@
+ALBU,552,292281.220,1005905.150
```

Comment:
As above, do we need this many test points?

Reply:
See above reply.